src/host/ut_host/Host.UnitTests.vcxproj

```diff
@@ -99,6 +99,15 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <AdditionalIncludeDirectories>..;$(SolutionDir)src\inc;$(SolutionDir)src\inc\test;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='AuditMode|ARM64'">/utf-8 %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='Debug|ARM64'">/utf-8 %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions Condition="'$(Configuration)|$(Platform)'=='Release|ARM64'">/utf-8 %(AdditionalOptions)</AdditionalOptions>

```

Comment:
There are like 200+ `C3437: #pragma execution_character_set is not supported when /source-charset, /execution-charset, or /utf-8 has been specified` errors jumping out when `/utf-8` is applied globally.  Most of them seem to result from macros like `TRACELOGGING_DEFINE_PROVIDER` and `TraceLogging*`.

Reply:
hrmm, I see what you mean. so it [sounds](https://developercommunity.visualstudio.com/content/problem/85934/traceloggingproviderh-is-incompatible-with-utf-8.html) like the tracelogging stuff doesn't play nice with the `/utf-8` flag. It looks like it's setting the [execution_character_set](https://docs.microsoft.com/en-us/cpp/preprocessor/execution-character-set?view=vs-2019) to be utf-8, which we want anyway. I'll need to do some more reading about what's going on here.

Reply:
Yeah, it's kind of self-contradictory that the Windows SDK, even the latest versions, uses a compiler directive that has been deprecated since VS 2015...